ompi/mca/coll/base/coll_base_bcast.c

```diff
@@ -655,27 +651,20 @@ ompi_coll_base_bcast_intra_basic_linear(void *buff, int count,
     }
 
     /* Root sends data to all others. */
-    preq = reqs = coll_base_comm_get_reqs(data, size-1);
+    preq = reqs = coll_base_comm_get_reqs(module->base_data, size-1);
```

Comment:
@bosilca  Missing the check on the pointer returned by coll_base_comm_get_reqs()